charts/vcluster/Chart.yaml

```diff
@@ -20,5 +20,5 @@ name: vcluster
 sources:
   - https://github.com/loft-sh/vcluster
 type: application
-version: 0.15.2
-appVersion: 0.15.2
+version: 0.0.0-dev
+appVersion: 0.0.0-dev
```

Comment:
we can leave the appVersion as the desired one, everything else is LGTM.

Reply:
done